### ISSUE TITLE
RadioButton adds extra padding

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#2450]
## Current behavior

RadioButton adds extra padding.

## Expected behavior

No padding

## How to reproduce it (as minimally and precisely as possible)

https://dev.azure.com/nventive/Umbrella/_workitems/edit/170847

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [x] iOS
- [x] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by carldebilly** [comment]
The problem is not related to the `<RadioButton>`, it's a layouting problem.

I reproduced the problem using the following snippet:
``` xml
<StackPanel Orientation="Horizontal" Spacing="10" VerticalAlignment="Center">
	<!-- You should not see the "light green" border (only yellow)
	     + both yellow & red should displays same height -->

	<Rectangle Fill="Red" Width="10" Height="45" VerticalAlignment="Top" />

	<Border Background="LightGreen" VerticalAlignment="Top">

		<RadioButton Height="40" MinHeight="45" Width="200" MinWidth="210">
			<RadioButton.Template>
				<ControlTemplate TargetType="RadioButton">
					<Grid Background="Yellow" />
				</ControlTemplate>
			</RadioButton.Template>
		</RadioButton>
	</Border>
</StackPanel>
```

**Comment by carldebilly** [comment]
This issue has been fixed by PR #2481
